pkgs/tools/misc/mongodb-tools/goconvey.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, goPackages, fetchFromGitHub, callPackage }:
```

Comment:
It didn't build with that other version of goconvey? Can you explain the problem?